cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -660,7 +660,9 @@ func TestCreateCoreDNSConfigMap(t *testing.T) {
         ttl 30
     }
     prometheus :9153
-    forward . /etc/resolv.conf
+    forward . /etc/resolv.conf {
+        max_concurrent 1000
```

Comment:
should we also mention this change in the release note? i don't have good statistical context on how breaking it is, but i'm going to assume it affects larger clusters with a lot of queries?

Reply:
The limit is per instance. The larger a cluster the more CoreDNS instances... So it should not change drastically at scale in theory.  For comparison, kube-dns (via it's default dnsmasq behavior), had a limit of 150 concurrent queries.  More details [here](https://github.com/coredns/deployment/pull/233#issue-438151578). CoreDNS never had a limit before this, it would just OOM if too many queries came in.

Reply:
ok, thanks for the details.

Reply:
Regarding release notes: CoreDNS 1.7.0 does introduce some backward incompatible changes. Notably, it makes many Prometheus metrics name changes (to bring them in line with standard Prometheus metrics naming convention).  They will be backward incompatible with existing reporting formulas that use the old metrics names.  Less critically, 1.7.0 drops support for a couple of kubernetes plugin options which had already been deprecated and ignored by CoreDNS for several releases (i.e. now instead of ignoring them, CoreDNS will exit with an unknown directive error).

Reply:
i think we should briefly mention that in the k8s release note and even link to the coredns release notes.

Reply:
Yes, I’ll update the release notes.